utils/src/test/resources/Person.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Alvin Alexander",
```

Comment:
Is this a random name? Lets just put name “Foo Bar”

Reply:
Yes, it is random. Updated to Foo Bar!